src/mito2/src/region/options.rs

```diff
@@ -152,6 +159,40 @@ impl Default for RegionOptionsWithoutEnum {
     }
 }
 
+with_prefix!(prefix_inverted_index "index.inverted_index.");
+
+/// Options for index.
+#[derive(Debug, Clone, PartialEq, Eq, Default, Deserialize)]
+#[serde(default)]
+pub struct IndexOptions {
+    /// Options for the inverted index.
+    #[serde(flatten, with = "prefix_inverted_index")]
+    pub inverted_index: InvertedIndexOptions,
+}
+
+/// Options for the inverted index.
+#[derive(Debug, Clone, PartialEq, Eq, Default, Deserialize)]
+#[serde(default)]
+pub struct InvertedIndexOptions {
+    /// The column ids that should be ignored when building the inverted index.
+    /// The column ids are separated by commas. For example, "1,2,3".
```

Comment:
Should we accept a json array `[1,2,3]` as input? Then we don't need to customize the deserialization method of this field.

Reply:
Custom deserialization should be unavoidable. The value type of options is String. There always needs to be a place to do this work.

My attempt:
```
0: Invalid options, at src/mito2/src/region/options.rs:71:71
1: Error("invalid type: string \"[1,2,3]\", expected a sequence", line: 1, column: 52)
```